Doc/library/typing.rst

```diff
@@ -2030,9 +2030,7 @@ Introspection helpers
 
    This is often the same as ``obj.__annotations__``. In addition,
    forward references encoded as string literals are handled by evaluating
-   them in ``globals`` and ``locals`` namespaces. If necessary,
```

Comment:
Add a versionchanged section mentioning the behavior change from previous versions.

Reply:
Done! To make bot happy: I have made the requested changes; please review again

Reply:
@sobolevn I think you need to post the message in a standalone message to make Bedevere happy, he's a pretty picky Knight :/